### PR TITLE
SCT-1267 Make Mash timeout configurable + more

### PR DESCRIPTION
**Pull request by MrCreosote**
Tested the mash timeout manually.

The dockerfile changes were contributed by @scanon.

Also some documentation fixes.

### CODE REVIEW

**Comment by MrCreosote** [comment]
Per @scanon, just bumping the minhash timeout fixed the timeout issues he was seeing, so I'll hold off on changing any of the jetty settings.

File: src/us/kbase/assemblyhomology/build/AssemblyHomologyBuilder.java [hunk 1]
```diff
@@ -74,7 +74,8 @@ private AssemblyHomology buildAssemblyHomology(
 		return new AssemblyHomology(
 				storage,
 				new HashSet<>(Arrays.asList(new MashFactory())),
-				c.getPathToTemporaryFileDirectory());
+				c.getPathToTemporaryFileDirectory(),
+				c.getMinhashTimoutSec());
```
Comment (jayrbolton) [inline src/us/kbase/assemblyhomology/build/AssemblyHomologyBuilder.java]: s/Timout/Timeout

Comment (MrCreosote) [reply src/us/kbase/assemblyhomology/build/AssemblyHomologyBuilder.java]: Fixed

**Comment by MrCreosote** [comment]
> Have you tried stress testing this? 

Lord no, this is KBase for God's sake.

Seriously though, no I haven't, but
a) I'm not particularly worried about it - I doubt this service is going to see many hits for a good while, and the caching service should be taking a good chunk of the load. The same Jetty server is powering the auth service which sits at the epicenter of KBase and has a much higher load - it's been DDOSed, practically, by badly coded SDK apps, and held up fine AFAIK.
b) If I had cycles to spend stress testing stuff it'd be the data stores. I'm way more worried about those than this service which sits waaaaay off to the side in KBase-land.

**Pull request merged**